wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -20,9 +20,10 @@ <h1 class="w-sr-only">
         {% if parent_page.get_latest_revision %}
             <a href="{% url 'wagtailadmin_pages:history' parent_page.id %}"
                 class="{{ nav_icon_button_classes }}"
-                data-tippy-content="{% trans 'History' %}"
-                data-tippy-offset="[0, 0]"
-                data-tippy-placement="bottom"
+                data-controller="w-tooltip"
+                data-w-tooltip-content-value="{% trans 'History' %}"
+                data-w-tooltip-offset="[0, 0]"
```

Comment:
```suggestion
                data-w-tooltip-offset-value="[0, 0]"
```

This will not work in its current form, you need to add the `-value` at the end. 

https://stimulus.hotwired.dev/reference/values

You have this right in the unit test but not in the html templates.